src/rootfs/files/configs/state_disk_mount/state_disk_mount.sh

```
@@ -0,0 +1,74 @@
+    exit 1;
+fi
+
+STATE_BLOCK_DEVICE_NAME="$(lsblk -d -n -o NAME | grep -v "$MAIN_BLOCK_DEVICE_NAME" | grep -v "$PROVIDER_CONFIG_BLOCK_DEVICE_NAME")";
+if [[ -z "$STATE_BLOCK_DEVICE_NAME" ]]; then
+    echo "Failed to get state block device, this error is abnomal and you should notify the SuperProtocol support team if you see this.."
+    exit 1;
+fi
+
+# Mounting encrypted state disk
+STATE_BLOCK_DEVICE_PATH="/dev/$STATE_BLOCK_DEVICE_NAME";
+RANDOM_KEY="$(dd if=/dev/urandom bs=1 count=32 2>/dev/null | base64)";
```

Shellcheck (0.10.0)

[warning] RANDOM_KEY appears unused. Verify use (or export if used externally).

(SC2034)